### PR TITLE
Add KIBANA_USERNAME/KIBANA_PASSWORD to agent container docs

### PR DESCRIPTION
**Pull request by michel-laterman**
Add missing env vars back to docs with updated description

### CODE REVIEW

**Comment by michel-laterman** [comment]
7.17/7.16 backport to update the deprecation notice: https://github.com/elastic/observability-docs/pull/1476

File: docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc:487]: There's really a default password? That's unexpected and probably not useful.

**Comment by dedemorton** [comment]
@elasticmachine run elasticsearch-ci/docs

**Review by dedemorton — APPROVED** [review]
Changes look OK, but the doc preview showed a 404, so I triggered a new docs build. Approving with the caveat that you (or I) must confirm the output in HTML before merging.

File: docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc
Comment (dedemorton) [inline docs/en/ingest-management/elastic-agent/configuration/env/shared-env.asciidoc:465]: Ah...the preview does not show up because you haven't included the tagged regions anywhere. You need to add include statements to any topics where you want the settings to appear. Look at container-envs.asciidoc for examples of how to include tagged regions.

**Review by dedemorton — CHANGES_REQUESTED** [review]
Found the problem with the preview! You'll need to fix this so the settings will appear in the published docs.

**Review by dedemorton — APPROVED** [review]
LGTM

**Pull request merged**
